portfolio/src/main/webapp/script.js

```diff
@@ -40,4 +40,22 @@ function slideShow(imgIndex = 0) {
   setTimeout(function() {
     slideShow(imgIndex);
   }, 5000);
+
+}
+
+/**
+ * Displays only courses that are part of a specified class.
+ */
+function filterCourseDisplay(courseType = 'course') {
+
+  const allCourses = document.getElementsByClassName('course');
+
+  // Display courses in courseType class and hide the rest.
+  for (let i = 0; i < allCourses.length; i++) {
+    if (allCourses[i].classList.contains(courseType)) {
+        allCourses[i].style.display = '';
+    } else {
+        allCourses[i].style.display = 'none';
+    }
```

Comment:
between for/foreach loops, prefer foreach for when you don't need the index of the list element: https://www.w3schools.com/js/tryit.asp?filename=tryjs_object_for_of